admin/app/tools/LoadBalancer.scala

```diff
@@ -38,16 +38,20 @@ object LoadBalancer {
   private val agent =  AkkaAgent(loadBalancers)
 
   def refresh() {
+    log.info("starting refresh LoadBalancer ELB DNS names")
     credentials.foreach{ credentials =>
       val client = new AmazonElasticLoadBalancingClient(credentials)
       client.setEndpoint(AwsEndpoints.elb)
-      val elbs = client.describeLoadBalancers().getLoadBalancerDescriptions
+      val request = new DescribeLoadBalancersRequest()
+      request.setLoadBalancerNames(loadBalancers.map(_.id))
```

Comment:
by doing this, if one of the ELBs is renamed or deleted it will break this job so I'm tempted to remove it